test/integration/tests/1198-multiple-exes-with-same-name/Main.hs

```diff
@@ -8,7 +8,7 @@ main = do
     stack [defaultResolverArg, "init", "--force"]
     stackCheckStderr
         ["build", "also-has-exe-foo", "has-exe-foo"]
-        (expectMessage buildMessage1)
+        (expectMessage buildMessage1) 
```

Comment:
There is no need to add an extra whitespace at the end of the line here, can you please remove it?

Reply:
👍